client/src/contexts/AuthContext.tsx

```diff
@@ -40,33 +35,27 @@ export const AuthProvider = ({
         }
       })
       .catch((reason: AxiosError) => {
-        // Catch 401 which signals an unauthorized user, which is not an issue
-        if (!(reason.response && reason.response.status === 401)) {
+        // Catch 400 or 401 which signals an unauthorized user, which is not an issue
```

Comment:
why would a 400 signal an unauthorized user? doesn't look like that status is returned by this endpoint, and if we're returning it then it doesn't seem like the right status code

Reply:
<img width="768" alt="image" src="https://user-images.githubusercontent.com/20250559/134123539-09cbbb1d-d3f8-4995-9740-2eaf3df69272.png">
You are right that this branch will not throw 400. The current develop branch will throw 400 if no auth. I am thinking of accepting both status first, then finally change to accept 401 only when the session is fully merged.

Reply:
Scratch that. I will update it to only accept 401 because we will be pushing this change when there are few users anyways.

Reply:
the new client will only ever see the new server and hence will never see 400s, so there's no point supporting both 400 and 401. lmk if that makes sense